### ISSUE TITLE
Customise jekyll theme

### ISSUE THREAD

**Issue by flanaras** [syfantid/Greek-CS-Student-Opportunities#2]
Having it as a markdown document in github doesn't seem very nice especially when dealing with big tables. Maybe it could be converted in a SOMETHING.github.io page or a different kind of view (list, etc) or link it on an external markdown viewer to have more screen space.

What are your thoughts on this?

**Comment by syfantid** [comment]
Agreed. Needing to scroll on tables is kind of annoying. Converting to GitHub page would be nice maybe. Have to check out the time it requires.

**Comment by syfantid** [comment]
What would you suggest instead of tables? So that it looks better and does not require scrolling?

**Comment by flanaras** [comment]
This should be done by the jekyll/css attributes.
